ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/subscription/blockheaders/NewBlockHeadersSubscriptionServiceTest.java

```diff
@@ -194,21 +244,31 @@ private void mockSubscriptionManagerNotifyMethod(
         .notifySubscribersOnWorkerThread(any(), any(), any());
   }
 
-  private void simulateAddingBlockOnCanonicalChain() {
-    final BlockBody blockBody = new BlockBody(Collections.emptyList(), Collections.emptyList());
-    final Block testBlock = new Block(blockHeader, blockBody);
+  private void simulateAddingBlockOnCanonicalChain(final Block block) {
     newBlockHeadersSubscriptionService.onBlockAdded(
-        BlockAddedEvent.createForHeadAdvancement(testBlock, Collections.emptyList()),
-        blockchainQueries.getBlockchain());
-    verify(blockchainQueries, times(1)).getBlockchain();
+        BlockAddedEvent.createForHeadAdvancement(block, Collections.emptyList()),
+        blockchainQueriesSpy.getBlockchain());
+    verify(blockchainQueriesSpy, times(1)).getBlockchain();
   }
 
   private void simulateAddingBlockOnNonCanonicalChain() {
     final BlockBody blockBody = new BlockBody(Collections.emptyList(), Collections.emptyList());
     final Block testBlock = new Block(blockHeader, blockBody);
```

Comment:
Now that we use block data generator, can we get rid of all the old fixtures and use the generator instead?